ddtrace/tracer/stats.go

```diff
@@ -65,6 +65,7 @@ type concentrator struct {
 func newConcentrator(c *config, bucketSize int64) *concentrator {
 	return &concentrator{
 		In:         make(chan *aggregableSpan, 10000),
+		wg:         sync.WaitGroup{},
```

Comment:
You can remove this. That's the main reason I liked the non-pointer one 🙂 There is no risk of `nil`. It's ready to use.

Reply:
Oh duh! The default value is safe to use, I misunderstood and thought you meant just remove the `&` :P